### PR TITLE
Update EN Doc: Integrate Harbor into Pipeline

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by Felixnoo** [comment]
/assign @LinuxSuRen Please review the updates about insecure registry I made to the doc [Integrate Harbor into Pipelines](https://kubesphere.io/docs/devops-user-guide/how-to-integrate/harbor/).

fix #1389

**Comment by Sherlock113** [comment]
@kubesphere/sig-devops

File: content/en/docs/devops-user-guide/how-to-integrate/harbor.md [hunk 1]
```diff
@@ -93,15 +124,15 @@ helm install harbor-release harbor/harbor --set expose.type=nodePort,externalURL
        // docker image name
        APP_NAME = 'docker-example'
        // ‘yuswift’ is the credential id you created on ks console
```
Comment (yuswift) [inline content/en/docs/devops-user-guide/how-to-integrate/harbor.md:126]: we should replace yuswift with robot-test

Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-integrate/harbor.md]: Good catch. Thanks for your comment.

**Pull request merged**